en/app.ftl

```diff
@@ -491,7 +493,9 @@ faq-question-disable-trackerremoval-answer = Yes. If you’re having trouble wit
 faq-question-bulk-trackerremoval-question = Can I remove trackers only on some of my email masks?
 faq-question-bulk-trackerremoval-answer = You can only turn tracker removal on at an account level — it either removes trackers from all of your emails, or none of them.
 faq-question-trackerremoval-breakage-question = Why do my emails look broken?
+# Deprecated
 faq-question-trackerremoval-breakage-answer = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
+faq-question-trackerremoval-breakage-answer-2 = Sometimes removing trackers may cause your email to look broken, because the trackers are often contained within images and links. When the tracker is removed, the email looks like it’s been formatted wrong because images are missing. This can’t be fixed for emails you’ve already received. If this is preventing you from reading your emails properly, turn off tracker removal.
```

Comment:
![image](https://user-images.githubusercontent.com/4251/182839459-42bc3e54-b17d-4288-8891-690eed3f1f6d.png)

and also

![image](https://user-images.githubusercontent.com/4251/182839522-ff6a6ba6-6d24-4dea-b925-046af7f8e532.png)

(The latter is an excerpt of the former (the full FAQ), so it makes sense that they're the same strings.)